packages/kbn-securitysolution-exceptions-common/scripts/openapi_bundle.js

```diff
@@ -46,6 +51,11 @@ const ROOT = resolve(__dirname, '..');
             "Exceptions API allows you to manage detection rule exceptions to prevent a rule from generating an alert from incoming events even when the rule's other criteria are met.",
         },
       },
+      assignTag: {
+        name: 'Security Solution Exceptions API',
+        description:
+          "Exceptions API allows you to manage detection rule exceptions to prevent a rule from generating an alert from incoming events even when the rule's other criteria are met.",
```

Comment:
The description is duplicated in two places: `prototypeDocument.description` and `assignTag.description`. What's the difference? Can we use a single property for the description?

Reply:
It's rather a question what happens to that information later on. We use `bundle()` to produce domain bundles and `merge()` to merge domain bundles together (not only Security Solution domain bundles). Domain bundle's `info.description` isn't included in the result Kibana bundle. But tags are included. Tag's description is shown on API reference documentation page.

That two descriptions don't have to match. It's good to have flexibility to specify different descriptions. Though `info.description` could used as tag's description when tag's description is omitted. WDYT?